### PR TITLE
Add sniff to ensure blank line before return statement.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: CakePHP/tests/files/blank_line_before_return_pass.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * [validFunctionReturnSix description]
+ *
+ * @return null|bool
+ */
+function validFunctionReturnSix()
+{
+    switch ($condition) {
+        case 'foo':
+            return true;
+
```
Comment (dereuromark) [inline CakePHP/tests/files/blank_line_before_return_pass.php]: Those newlines don't need to be imo

Comment (ADmad) [inline CakePHP/tests/files/blank_line_before_return_pass.php]: This sniff only checks for new line *before* the return statement. New line after the statement are irrelevant.

**Comment by ADmad** [comment]
@markstory I was pretty sure one existed, just had to check whether it was smart enough. :smile:

**Comment by markstory** [comment]
Any idea on how much work it will take to make this sniff pass on the core code?

**Comment by antograssiot** [comment]
something like

```
PHP CODE SNIFFER REPORT SUMMARY
----------------------------------------------------------------------
FILE                                                  ERRORS  WARNINGS
----------------------------------------------------------------------
...ites/github/cakephp/src/Auth/BaseAuthenticate.php  1       0
...tes/github/cakephp/src/Auth/BasicAuthenticate.php  2       0
...s/github/cakephp/src/Auth/ControllerAuthorize.php  1       0
...es/github/cakephp/src/Auth/DigestAuthenticate.php  4       0
...ithub/cakephp/src/Auth/FallbackPasswordHasher.php  1       0
...ites/github/cakephp/src/Auth/FormAuthenticate.php  2       0
...github/cakephp/src/Auth/Storage/MemoryStorage.php  1       0
...ithub/cakephp/src/Auth/Storage/SessionStorage.php  2       0
.../antograssiot/Sites/github/cakephp/src/basics.php  1       0
...grassiot/Sites/github/cakephp/src/Cache/Cache.php  11      0
...ot/Sites/github/cakephp/src/Cache/CacheEngine.php  5       0
.../Sites/github/cakephp/src/Cache/CacheRegistry.php  2       0
...tes/github/cakephp/src/Cache/Engine/ApcEngine.php  8       0
...es/github/cakephp/src/Cache/Engine/FileEngine.php  10      0
...thub/cakephp/src/Cache/Engine/MemcachedEngine.php  5       0
...s/github/cakephp/src/Cache/Engine/RedisEngine.php  5       0
...ithub/cakephp/src/Cache/Engine/WincacheEngine.php  6       0
.../github/cakephp/src/Cache/Engine/XcacheEngine.php  6       0
...github/cakephp/src/Collection/CollectionTrait.php  21      0
...es/github/cakephp/src/Collection/ExtractTrait.php  4       0
...ephp/src/Collection/Iterator/BufferedIterator.php  3       0
...kephp/src/Collection/Iterator/ExtractIterator.php  1       0
...akephp/src/Collection/Iterator/InsertIterator.php  1       0
...hub/cakephp/src/Collection/Iterator/MapReduce.php  1       0
.../cakephp/src/Collection/Iterator/NestIterator.php  1       0
...kephp/src/Collection/Iterator/ReplaceIterator.php  1       0
...php/src/Collection/Iterator/StoppableIterator.php  1       0
.../cakephp/src/Collection/Iterator/TreeIterator.php  1       0
...b/cakephp/src/Collection/Iterator/TreePrinter.php  3       0
...Sites/github/cakephp/src/Console/ConsoleInput.php  3       0
...thub/cakephp/src/Console/ConsoleInputArgument.php  4       0
...github/cakephp/src/Console/ConsoleInputOption.php  4       0
...ub/cakephp/src/Console/ConsoleInputSubcommand.php  3       0
...ot/Sites/github/cakephp/src/Console/ConsoleIo.php  6       0
...ithub/cakephp/src/Console/ConsoleOptionParser.php  26      0
...ites/github/cakephp/src/Console/ConsoleOutput.php  6       0
...ites/github/cakephp/src/Console/HelpFormatter.php  4       0
...assiot/Sites/github/cakephp/src/Console/Shell.php  22      0
...es/github/cakephp/src/Console/ShellDispatcher.php  9       0
...akephp/src/Controller/Component/AuthComponent.php  23      0
...ephp/src/Controller/Component/CookieComponent.php  4       0
...p/src/Controller/Component/PaginatorComponent.php  6       0
.../Controller/Component/RequestHandlerComponent.php  14      0
...hp/src/Controller/Component/SecurityComponent.php  12      0
...Sites/github/cakephp/src/Controller/Component.php  1       0
...thub/cakephp/src/Controller/ComponentRegistry.php  1       0
...ites/github/cakephp/src/Controller/Controller.php  11      0
...ntograssiot/Sites/github/cakephp/src/Core/App.php  2       0
...iot/Sites/github/cakephp/src/Core/ClassLoader.php  2       0
...b/cakephp/src/Core/Configure/Engine/IniConfig.php  4       0
.../cakephp/src/Core/Configure/Engine/JsonConfig.php  2       0
...b/cakephp/src/Core/Configure/Engine/PhpConfig.php  1       0
...ssiot/Sites/github/cakephp/src/Core/Configure.php  14      0
...ites/github/cakephp/src/Core/ConventionsTrait.php  3       0
...ssiot/Sites/github/cakephp/src/Core/functions.php  8       0
...s/github/cakephp/src/Core/InstanceConfigTrait.php  7       0
.../Sites/github/cakephp/src/Core/ObjectRegistry.php  4       0
...grassiot/Sites/github/cakephp/src/Core/Plugin.php  9       0
...tes/github/cakephp/src/Core/StaticConfigTrait.php  3       0
.../Sites/github/cakephp/src/Database/Connection.php  17      0
...akephp/src/Database/Dialect/MysqlDialectTrait.php  1       0
...php/src/Database/Dialect/PostgresDialectTrait.php  4       0
...kephp/src/Database/Dialect/SqliteDialectTrait.php  4       0
...hp/src/Database/Dialect/SqlserverDialectTrait.php  7       0
...tabase/Dialect/TupleComparisonTranslatorTrait.php  2       0
...ites/github/cakephp/src/Database/Driver/Mysql.php  2       0
...ub/cakephp/src/Database/Driver/PDODriverTrait.php  10      0
...s/github/cakephp/src/Database/Driver/Postgres.php  1       0
...tes/github/cakephp/src/Database/Driver/Sqlite.php  2       0
.../github/cakephp/src/Database/Driver/Sqlserver.php  2       0
...siot/Sites/github/cakephp/src/Database/Driver.php  3       0
...php/src/Database/Expression/BetweenExpression.php  1       0
...ub/cakephp/src/Database/Expression/Comparison.php  1       0
...hp/src/Database/Expression/FunctionExpression.php  2       0
...php/src/Database/Expression/OrderByExpression.php  1       0
...src/Database/Expression/OrderClauseExpression.php  1       0
...kephp/src/Database/Expression/QueryExpression.php  23      0
...kephp/src/Database/Expression/TupleComparison.php  3       0
...ephp/src/Database/Expression/ValuesExpression.php  4       0
...ithub/cakephp/src/Database/FieldTypeConverter.php  1       0
.../github/cakephp/src/Database/FunctionsBuilder.php  4       0
.../github/cakephp/src/Database/IdentifierQuoter.php  7       0
...hub/cakephp/src/Database/Log/LoggingStatement.php  2       0
...s/github/cakephp/src/Database/Log/QueryLogger.php  1       0
...ssiot/Sites/github/cakephp/src/Database/Query.php  44      0
...tes/github/cakephp/src/Database/QueryCompiler.php  10      0
...github/cakephp/src/Database/Schema/BaseSchema.php  2       0
.../cakephp/src/Database/Schema/CachedCollection.php  1       0
...github/cakephp/src/Database/Schema/Collection.php  1       0
...ithub/cakephp/src/Database/Schema/MysqlSchema.php  9       0
...ub/cakephp/src/Database/Schema/PostgresSchema.php  14      0
...thub/cakephp/src/Database/Schema/SqliteSchema.php  9       0
...b/cakephp/src/Database/Schema/SqlserverSchema.php  12      0
...ites/github/cakephp/src/Database/Schema/Table.php  21      0
...s/github/cakephp/src/Database/SqlDialectTrait.php  4       0
...github/cakephp/src/Database/SqlserverCompiler.php  1       0
...ephp/src/Database/Statement/BufferedStatement.php  5       0
...php/src/Database/Statement/BufferResultsTrait.php  1       0
...ephp/src/Database/Statement/CallbackStatement.php  1       0
...cakephp/src/Database/Statement/MysqlStatement.php  1       0
...b/cakephp/src/Database/Statement/PDOStatement.php  2       0
...akephp/src/Database/Statement/SqliteStatement.php  2       0
...php/src/Database/Statement/StatementDecorator.php  3       0
...tes/github/cakephp/src/Database/Type/BoolType.php  2       0
...github/cakephp/src/Database/Type/DateTimeType.php  8       0
...tes/github/cakephp/src/Database/Type/DateType.php  5       0
...es/github/cakephp/src/Database/Type/FloatType.php  5       0
.../github/cakephp/src/Database/Type/IntegerType.php  2       0
...s/github/cakephp/src/Database/Type/StringType.php  2       0
...tes/github/cakephp/src/Database/Type/TimeType.php  1       0
...tes/github/cakephp/src/Database/Type/UuidType.php  1       0
...assiot/Sites/github/cakephp/src/Database/Type.php  5       0
...ithub/cakephp/src/Database/TypeConverterTrait.php  2       0
.../github/cakephp/src/Database/TypedResultTrait.php  1       0
...iot/Sites/github/cakephp/src/Database/TypeMap.php  3       0
...ites/github/cakephp/src/Database/TypeMapTrait.php  2       0
...Sites/github/cakephp/src/Database/ValueBinder.php  1       0
...thub/cakephp/src/Datasource/ConnectionManager.php  3       0
...hub/cakephp/src/Datasource/ConnectionRegistry.php  2       0
...tes/github/cakephp/src/Datasource/EntityTrait.php  21      0
...github/cakephp/src/Datasource/ModelAwareTrait.php  1       0
...tes/github/cakephp/src/Datasource/QueryCacher.php  4       0
...ites/github/cakephp/src/Datasource/QueryTrait.php  11      0
...tes/github/cakephp/src/Datasource/RuleInvoker.php  2       0
...github/cakephp/src/Datasource/RulesAwareTrait.php  2       0
...es/github/cakephp/src/Datasource/RulesChecker.php  6       0
...tes/github/cakephp/src/Error/BaseErrorHandler.php  7       0
...ssiot/Sites/github/cakephp/src/Error/Debugger.php  12      0
...t/Sites/github/cakephp/src/Error/ErrorHandler.php  1       0
...es/github/cakephp/src/Error/ExceptionRenderer.php  12      0
...github/cakephp/src/Event/EventDispatcherTrait.php  1       0
...siot/Sites/github/cakephp/src/Event/EventList.php  2       0
...t/Sites/github/cakephp/src/Event/EventManager.php  19      0
...siot/Sites/github/cakephp/src/Filesystem/File.php  23      0
...ot/Sites/github/cakephp/src/Filesystem/Folder.php  30      0
...tograssiot/Sites/github/cakephp/src/Form/Form.php  5       0
...grassiot/Sites/github/cakephp/src/Form/Schema.php  5       0
...s/github/cakephp/src/I18n/ChainMessagesLoader.php  1       0
...Sites/github/cakephp/src/I18n/DateFormatTrait.php  7       0
...siot/Sites/github/cakephp/src/I18n/FrozenTime.php  4       0
...ssiot/Sites/github/cakephp/src/I18n/functions.php  8       0
...tograssiot/Sites/github/cakephp/src/I18n/I18n.php  4       0
...es/github/cakephp/src/I18n/MessagesFileLoader.php  1       0
...grassiot/Sites/github/cakephp/src/I18n/Number.php  8       0
...s/github/cakephp/src/I18n/Parser/MoFileParser.php  1       0
...github/cakephp/src/I18n/RelativeTimeFormatter.php  7       0
...tograssiot/Sites/github/cakephp/src/I18n/Time.php  4       0
...es/github/cakephp/src/I18n/TranslatorRegistry.php  5       0
...ites/github/cakephp/src/Log/Engine/ConsoleLog.php  1       0
...s/github/cakephp/src/Mailer/AbstractTransport.php  1       0
...rassiot/Sites/github/cakephp/src/Mailer/Email.php  49      0
...assiot/Sites/github/cakephp/src/Mailer/Mailer.php  5       0
...b/cakephp/src/Mailer/Transport/DebugTransport.php  1       0
...ub/cakephp/src/Mailer/Transport/MailTransport.php  1       0
...ub/cakephp/src/Mailer/Transport/SmtpTransport.php  4       0
.../Sites/github/cakephp/src/Network/CorsBuilder.php  8       0
...ithub/cakephp/src/Network/Http/Adapter/Stream.php  5       0
...s/github/cakephp/src/Network/Http/Auth/Digest.php  2       0
...es/github/cakephp/src/Network/Http/Auth/Oauth.php  5       0
.../Sites/github/cakephp/src/Network/Http/Client.php  14      0
...hub/cakephp/src/Network/Http/CookieCollection.php  1       0
...github/cakephp/src/Network/Http/FormData/Part.php  1       0
...ites/github/cakephp/src/Network/Http/FormData.php  7       0
...Sites/github/cakephp/src/Network/Http/Message.php  2       0
...Sites/github/cakephp/src/Network/Http/Request.php  6       0
...ites/github/cakephp/src/Network/Http/Response.php  12      0
...siot/Sites/github/cakephp/src/Network/Request.php  48      0
...iot/Sites/github/cakephp/src/Network/Response.php  32      0
...thub/cakephp/src/Network/Session/CacheSession.php  1       0
...b/cakephp/src/Network/Session/DatabaseSession.php  2       0
...siot/Sites/github/cakephp/src/Network/Session.php  4       0
...ssiot/Sites/github/cakephp/src/Network/Socket.php  14      0
.../github/cakephp/src/ORM/Association/BelongsTo.php  5       0
...hub/cakephp/src/ORM/Association/BelongsToMany.php  21      0
...ephp/src/ORM/Association/DependentDeleteTrait.php  2       0
.../src/ORM/Association/ExternalAssociationTrait.php  5       0
...es/github/cakephp/src/ORM/Association/HasMany.php  7       0
...tes/github/cakephp/src/ORM/Association/HasOne.php  5       0
...rc/ORM/Association/SelectableAssociationTrait.php  10      0
...siot/Sites/github/cakephp/src/ORM/Association.php  19      0
.../github/cakephp/src/ORM/AssociationCollection.php  9       0
...php/src/ORM/Behavior/Translate/TranslateTrait.php  1       0
...ub/cakephp/src/ORM/Behavior/TranslateBehavior.php  4       0
.../github/cakephp/src/ORM/Behavior/TreeBehavior.php  13      0
...rassiot/Sites/github/cakephp/src/ORM/Behavior.php  3       0
...Sites/github/cakephp/src/ORM/BehaviorRegistry.php  6       0
...ot/Sites/github/cakephp/src/ORM/EagerLoadable.php  1       0
...siot/Sites/github/cakephp/src/ORM/EagerLoader.php  9       0
...ograssiot/Sites/github/cakephp/src/ORM/Entity.php  1       0
.../Sites/github/cakephp/src/ORM/LazyEagerLoader.php  3       0
...hub/cakephp/src/ORM/Locator/LocatorAwareTrait.php  1       0
...s/github/cakephp/src/ORM/Locator/TableLocator.php  3       0
...ssiot/Sites/github/cakephp/src/ORM/Marshaller.php  11      0
...tograssiot/Sites/github/cakephp/src/ORM/Query.php  22      0
...assiot/Sites/github/cakephp/src/ORM/ResultSet.php  5       0
...ot/Sites/github/cakephp/src/ORM/Rule/ExistsIn.php  2       0
...ot/Sites/github/cakephp/src/ORM/Rule/IsUnique.php  1       0
...iot/Sites/github/cakephp/src/ORM/RulesChecker.php  3       0
...tograssiot/Sites/github/cakephp/src/ORM/Table.php  34      0
...t/Sites/github/cakephp/src/Routing/Dispatcher.php  1       0
.../github/cakephp/src/Routing/DispatcherFactory.php  3       0
...s/github/cakephp/src/Routing/DispatcherFilter.php  1       0
...github/cakephp/src/Routing/Filter/AssetFilter.php  2       0
...hp/src/Routing/Filter/ControllerFactoryFilter.php  1       0
...thub/cakephp/src/Routing/Filter/RoutingFilter.php  1       0
...github/cakephp/src/Routing/RequestActionTrait.php  1       0
.../github/cakephp/src/Routing/Route/DashedRoute.php  4       0
...thub/cakephp/src/Routing/Route/InflectedRoute.php  3       0
...ub/cakephp/src/Routing/Route/PluginShortRoute.php  2       0
.../Sites/github/cakephp/src/Routing/Route/Route.php  14      0
...Sites/github/cakephp/src/Routing/RouteBuilder.php  2       0
...es/github/cakephp/src/Routing/RouteCollection.php  1       0
...ssiot/Sites/github/cakephp/src/Routing/Router.php  19      0
...tes/github/cakephp/src/Shell/CommandListShell.php  1       0
...ites/github/cakephp/src/Shell/CompletionShell.php  2       0
...s/github/cakephp/src/Shell/Helper/TableHelper.php  1       0
...siot/Sites/github/cakephp/src/Shell/I18nShell.php  1       0
.../Sites/github/cakephp/src/Shell/OrmCacheShell.php  4       0
...ot/Sites/github/cakephp/src/Shell/RoutesShell.php  4       0
...ites/github/cakephp/src/Shell/Task/AssetsTask.php  6       0
...tes/github/cakephp/src/Shell/Task/CommandTask.php  3       0
...tes/github/cakephp/src/Shell/Task/ExtractTask.php  8       0
.../Sites/github/cakephp/src/Shell/Task/LoadTask.php  3       0
...ites/github/cakephp/src/Shell/Task/UnloadTask.php  2       0
.../cakephp/src/TestSuite/Fixture/FixtureManager.php  1       0
...hub/cakephp/src/TestSuite/Fixture/TestFixture.php  9       0
...hub/cakephp/src/TestSuite/IntegrationTestCase.php  5       0
...t/Sites/github/cakephp/src/TestSuite/TestCase.php  5       0
...s/github/cakephp/src/Utility/CookieCryptTrait.php  5       0
...ites/github/cakephp/src/Utility/Crypto/Mcrypt.php  3       0
...tes/github/cakephp/src/Utility/Crypto/OpenSsl.php  2       0
...rassiot/Sites/github/cakephp/src/Utility/Hash.php  29      0
...ot/Sites/github/cakephp/src/Utility/Inflector.php  11      0
...ithub/cakephp/src/Utility/MergeVariablesTrait.php  1       0
...iot/Sites/github/cakephp/src/Utility/Security.php  7       0
...rassiot/Sites/github/cakephp/src/Utility/Text.php  16      0
...grassiot/Sites/github/cakephp/src/Utility/Xml.php  3       0
...s/github/cakephp/src/Validation/RulesProvider.php  1       0
...ites/github/cakephp/src/Validation/Validation.php  38      0
.../github/cakephp/src/Validation/ValidationRule.php  3       0
...s/github/cakephp/src/Validation/ValidationSet.php  4       0
...Sites/github/cakephp/src/Validation/Validator.php  63      0
...ub/cakephp/src/Validation/ValidatorAwareTrait.php  1       0
...tograssiot/Sites/github/cakephp/src/View/Cell.php  3       0
...ssiot/Sites/github/cakephp/src/View/CellTrait.php  2       0
...tes/github/cakephp/src/View/Form/ArrayContext.php  10      0
...es/github/cakephp/src/View/Form/EntityContext.php  14      0
...ites/github/cakephp/src/View/Form/FormContext.php  3       0
...tes/github/cakephp/src/View/Helper/FormHelper.php  48      0
...tes/github/cakephp/src/View/Helper/HtmlHelper.php  20      0
...thub/cakephp/src/View/Helper/IdGeneratorTrait.php  2       0
...s/github/cakephp/src/View/Helper/NumberHelper.php  3       0
...ithub/cakephp/src/View/Helper/PaginatorHelper.php  21      0
...ites/github/cakephp/src/View/Helper/RssHelper.php  4       0
...tes/github/cakephp/src/View/Helper/TextHelper.php  7       0
...tes/github/cakephp/src/View/Helper/TimeHelper.php  7       0
...ites/github/cakephp/src/View/Helper/UrlHelper.php  7       0
...grassiot/Sites/github/cakephp/src/View/Helper.php  4       0
.../Sites/github/cakephp/src/View/HelperRegistry.php  3       0
...assiot/Sites/github/cakephp/src/View/JsonView.php  2       0
.../Sites/github/cakephp/src/View/SerializedView.php  1       0
.../Sites/github/cakephp/src/View/StringTemplate.php  4       0
...s/github/cakephp/src/View/StringTemplateTrait.php  2       0
...tograssiot/Sites/github/cakephp/src/View/View.php  18      0
...ssiot/Sites/github/cakephp/src/View/ViewBlock.php  4       0
...iot/Sites/github/cakephp/src/View/ViewBuilder.php  13      0
...t/Sites/github/cakephp/src/View/ViewVarsTrait.php  3       0
...es/github/cakephp/src/View/Widget/BasicWidget.php  1       0
...s/github/cakephp/src/View/Widget/ButtonWidget.php  1       0
...github/cakephp/src/View/Widget/CheckboxWidget.php  1       0
...github/cakephp/src/View/Widget/DateTimeWidget.php  11      0
...tes/github/cakephp/src/View/Widget/FileWidget.php  1       0
...b/cakephp/src/View/Widget/MultiCheckboxWidget.php  4       0
...es/github/cakephp/src/View/Widget/RadioWidget.php  3       0
...ithub/cakephp/src/View/Widget/SelectBoxWidget.php  7       0
...github/cakephp/src/View/Widget/TextareaWidget.php  1       0
...github/cakephp/src/View/Widget/WidgetRegistry.php  2       0
...rassiot/Sites/github/cakephp/src/View/XmlView.php  1       0
...cakephp/tests/test_app/config/htmlhelper_tags.php  1       0
...github/cakephp/tests/test_app/config/var_test.php  1       0
...ithub/cakephp/tests/test_app/config/var_test2.php  1       0
...t_app/Plugin/TestPlugin/config/test_templates.php  1       0
...gin/TestPlugin/src/Controller/TestsController.php  1       0
...ugin/src/Routing/Filter/Test2DispatcherFilter.php  1       0
...est_app/TestApp/Controller/AuthTestController.php  1       0
...estApp/Controller/Component/TestAuthComponent.php  1       0
...s/test_app/TestApp/Controller/PostsController.php  2       0
...pp/TestApp/Controller/RequestActionController.php  10      0
...st_app/TestApp/Controller/SomePagesController.php  1       0
...st_app/TestApp/Controller/TestsAppsController.php  2       0
...b/cakephp/tests/test_app/TestApp/Core/TestApp.php  1       0
...t_app/TestApp/Error/TestAppsExceptionRenderer.php  1       0
..._app/TestApp/Model/Behavior/SluggableBehavior.php  2       0
...sts/test_app/TestApp/Model/Table/AuthorsTable.php  1       0
...t_app/TestApp/Model/Table/PaginatorPostsTable.php  3       0
...ts/test_app/TestApp/Routing/Route/DashedRoute.php  1       0
.../tests/test_app/TestApp/Shell/Task/SampleTask.php  1       0
...ephp/tests/TestCase/Collection/CollectionTest.php  3       0
...stCase/Controller/Component/AuthComponentTest.php  1       0
...Case/Controller/Component/CookieComponentTest.php  2       0
...e/Controller/Component/PaginatorComponentTest.php  2       0
...se/Controller/Component/SecurityComponentTest.php  1       0
...es/github/cakephp/tests/TestCase/Core/AppTest.php  1       0
...php/tests/TestCase/Core/StaticConfigTraitTest.php  1       0
...akephp/tests/TestCase/Database/ConnectionTest.php  3       0
...hub/cakephp/tests/TestCase/Database/QueryTest.php  16      0
...ests/TestCase/Database/Schema/MysqlSchemaTest.php  1       0
...s/TestCase/Database/Schema/PostgresSchemaTest.php  1       0
...sts/TestCase/Database/Schema/SqliteSchemaTest.php  1       0
.../TestCase/Database/Schema/SqlserverSchemaTest.php  1       0
...s/github/cakephp/tests/TestCase/DatabaseSuite.php  2       0
...sts/TestCase/Datasource/ConnectionManagerTest.php  1       0
...tests/TestCase/Datasource/ModelAwareTraitTest.php  2       0
...php/tests/TestCase/Datasource/QueryCacherTest.php  1       0
...hp/tests/TestCase/Error/ExceptionRendererTest.php  2       0
...ub/cakephp/tests/TestCase/Filesystem/FileTest.php  1       0
...s/github/cakephp/tests/TestCase/I18n/I18nTest.php  14      0
...p/tests/TestCase/I18n/Parser/PoFileParserTest.php  1       0
...tes/github/cakephp/tests/TestCase/Log/LogTest.php  1       0
...ithub/cakephp/tests/TestCase/Mailer/EmailTest.php  1       0
...s/TestCase/Mailer/Transport/SmtpTransportTest.php  1       0
...ests/TestCase/Network/Http/Adapter/StreamTest.php  1       0
...b/cakephp/tests/TestCase/Network/ResponseTest.php  1       0
...ts/TestCase/ORM/Association/BelongsToManyTest.php  2       0
...estCase/ORM/Behavior/CounterCacheBehaviorTest.php  1       0
...s/TestCase/ORM/Behavior/TranslateBehaviorTest.php  2       0
...ub/cakephp/tests/TestCase/ORM/EagerLoaderTest.php  2       0
.../github/cakephp/tests/TestCase/ORM/EntityTest.php  5       0
...hub/cakephp/tests/TestCase/ORM/MarshallerTest.php  2       0
...akephp/tests/TestCase/ORM/QueryRegressionTest.php  2       0
...s/github/cakephp/tests/TestCase/ORM/QueryTest.php  10      0
...ests/TestCase/ORM/RulesCheckerIntegrationTest.php  7       0
...s/github/cakephp/tests/TestCase/ORM/TableTest.php  2       0
...p/tests/TestCase/Routing/DispatcherFilterTest.php  1       0
...cakephp/tests/TestCase/Routing/DispatcherTest.php  1       0
...hub/cakephp/tests/TestCase/Routing/RouterTest.php  2       0
.../tests/TestCase/Validation/ValidationRuleTest.php  2       0
...kephp/tests/TestCase/Validation/ValidatorTest.php  8       0
...php/tests/TestCase/View/Helper/FormHelperTest.php  1       0
...tests/TestCase/View/Widget/DateTimeWidgetTest.php  1       0
----------------------------------------------------------------------
A TOTAL OF 1951 ERRORS AND 0 WARNINGS WERE FOUND IN 340 FILES
----------------------------------------------------------------------
```

still +1 for me

**Comment by ADmad** [comment]
@markstory I will take care of updating core code once this is merged.

**Comment by markstory** [comment]
@ADmad Go for it then.

**Pull request merged**